tests/integration/runner

```diff
@@ -188,7 +188,8 @@ if __name__ == "__main__":
     cmd = "docker run {net} {tty} --rm --name {name} --privileged --volume={bridge_bin}:/clickhouse-odbc-bridge --volume={bin}:/clickhouse \
         --volume={base_cfg}:/clickhouse-config --volume={cases_dir}:/ClickHouse/tests/integration \
         --volume={src_dir}/Server/grpc_protos:/ClickHouse/src/Server/grpc_protos \
-        --volume={name}_volume:/var/lib/docker {env_tags} -e PYTEST_OPTS='{opts}' {img} {command}".format(
+        --volume={name}_volume:/var/lib/docker \
+        --volume=/etc/docker:/etc/docker {env_tags} -e PYTEST_OPTS='{opts}' {img} {command}".format(
```

Comment:
You want to share this directory between two docker daemons? Are you sure it works correctly considering we run docker in docker?

Reply:
> Are you sure it works correctly considering we run docker in docker?

We already did that. This PR only add one more sharing : the docker configuration file.